shell/components/Resource/Detail/ResourcePopover/ResourcePopoverCard.vue

```diff
@@ -0,0 +1,144 @@
+<script lang="ts">
+import Card from '@shell/components/Resource/Detail/Card/index.vue';
+import { useStore } from 'vuex';
+import ActionMenu from '@shell/components/ActionMenuShell.vue';
+import { useI18n } from '@shell/composables/useI18n';
+
+export interface Props {
+  resource: any;
+}
+</script>
+
+<script setup lang="ts">
+const emit = defineEmits(['action-invoked']);
+const props = defineProps<Props>();
+const store = useStore();
+const i18n = useI18n(store);
+
+const getGlanceItemValueId = (glanceItem: any): string => `value-${ glanceItem.label }:${ glanceItem.content }`.toLowerCase().replaceAll(' ', '');
+</script>
+
+<template>
+  <Card
+    class="resource-popover-card"
+    :title="'cattle-system'"
+    dropdown-menu-item
```

Comment:
The `dropdown-menu-item` attribute should only be required for RcDropdown menu items.

Reply:
Correct, left over from an early implementation.

Removed.